scipy/optimize/tests/test__differential_evolution.py

```diff
@@ -1339,7 +1339,7 @@ def c1(x):
         assert_(np.all(res.x >= np.array(bounds)[:, 0]))
         assert_(np.all(res.x <= np.array(bounds)[:, 1]))
 
-    @pytest.mark.slow
+    @pytest.mark.xslow
```

Comment:
This is just one of many tests. This one happens to be very slow, and it fails on some platforms, so probably no harm in making it xslow.